src/explorer-ui.js

```diff
@@ -7,8 +7,7 @@ import { LocationContext } from './location';
 const ExplorerUI = () => {
   const { locationUrl, document, onEntryPoint } = useContext(LocationContext);
   const [entryPointLinks, setEntryPointLinks] = useState({});
-  const parsedLinks =
-    document && document.links ? Link.parseLinks(document.links) : {};
+  const parsedLinks = document ? document.getLinks() : {};
```

Comment:
This is technically valid but it's a little confusing to have methods like this- since it looks like you're referring to the global "document" as in `document.getElementById()`

Reply:
I renamed this variable to `responseDocument`. WDYT?

Reply:
I think that's better, there's a few other places where you use `document` like location:47, resource:19,20, schema-parser:64,70,84, schema:10

Reply:
Renamed all similar occurrences.